runtime/rasdump/dmpsup.c

```diff
@@ -589,6 +624,13 @@ configureDumpAgents(J9JavaVM *vm)
 					agentOpts[agentNum].pass = J9RAS_DUMP_OPTS_PASS_ONE;
 					agentNum++;
 				}
+			} else if (isMappedToolDump) {
+				char * toolString = "tool";
+				agentOpts[agentNum].kind = scanDumpType(&toolString);
+				agentOpts[agentNum].flags = J9RAS_DUMP_OPT_ARGS_ALLOC;
```

Comment:
@keithc-ca 
Here's the modified implementation based on our discussion, using the existing flag `J9RAS_DUMP_OPT_ARGS_ALLOC`. The memory allocation for `optionString` no longer includes "tool:".